src/libraries/System.Text.Json/src/System/Text/Json/Document/JsonDocument.cs

```diff
@@ -363,6 +379,12 @@ internal string GetNameOfPropertyValue(int index)
             return GetString(index - DbRow.Size, JsonTokenType.PropertyName)!;
         }
 
+        internal ReadOnlySpan<byte> GetRawNameOfPropertyValue(int index)
```

Comment:
This looks like a stub method. Would it be possible to just inline its logic in the `JsonElement.GetRawPropertyName` method?